code/game/gamemodes/changeling/powers/biodegrade.dm

```diff
@@ -0,0 +1,82 @@
+/obj/effect/proc_holder/changeling/biodegrade
+	name = "Biodegrade"
+	desc = "Dissolves restraints or other objects preventing free movement."
+	helptext = "This is obvious to nearby people, and can destroy standard restraints and closets."
+	chemical_cost = 30 //High cost to prevent spam
+	dna_cost = 2
+	req_human = 1
+
+/obj/effect/proc_holder/changeling/biodegrade/sting_action(mob/living/carbon/human/user)
+	var/used = FALSE // only one form of shackles removed per use
+	if(!user.restrained() && !istype(user.loc, /obj/structure/closet) && !istype(user.loc, /obj/structure/spider/cocoon))
+		to_chat(user, "<span class='warning'>We are already free!</span>")
```

Comment:
"We"? I wasn't aware changelings talk in the, what is that third person? Besides that I thought all those notifications were just that like "You are already free"

Reply:
We is used on other changeling spells also.

Reply:
That's weird. Changelings are still one being, just with multiple faces. I guess they do get some of the previous memories of the person they suck. I must be missing something about changelings

Reply:
Hivemind, meaning a single changeling is still part of a collective, hence the use of the inclusive collective pronouns. "We must absorb Baldie McGriffin", "We reconfigure ourselves", etc